node/src/chain_spec.rs

```diff
@@ -564,6 +580,85 @@ fn amplitude_genesis(
 			members: signatories.clone(),
 			..Default::default()
 		},
+		tokens: amplitude_runtime::TokensConfig {
+			// Configure the initial token supply for the native currency and USDC asset
+			balances: token_balances,
+		},
+		issue: amplitude_runtime::IssueConfig {
+			issue_period: amplitude_runtime::DAYS,
+			issue_minimum_transfer_amount: 1000,
+			limit_volume_amount: None,
+			limit_volume_currency_id: XCM(DOT),
```

Comment:
Just leaving this comment as a reminder that we want to change this.

Reply:
done